### PR TITLE
update Coder railway template link

### PR DESCRIPTION
**Pull request by matifali**
This PR updates the Coder's Railway.app template link.

we now have https://railway.app/template/coder

### CODE REVIEW

**Review by ammario — APPROVED** [review]

**Comment by ammario** [comment]
Feel free to merge into this repo without review, esp. with minor changes such as this.

**Comment by matifali** [comment]
Can't merge without approval. It doesn't allow it.

**Comment by ammario** [comment]
@matifali — fixed

**Pull request merged**
